pkg/volume/azure_file/azure_file.go

```diff
@@ -222,11 +222,12 @@ func (b *azureFileMounter) SetUpAt(dir string, fsGroup *int64) error {
 	} else {
 		os.MkdirAll(dir, 0700)
 		// parameters suggested by https://azure.microsoft.com/en-us/documentation/articles/storage-how-to-use-files-linux/
-		options := []string{fmt.Sprintf("vers=3.0,username=%s,password=%s,dir_mode=0700,file_mode=0700", accountName, accountKey)}
```

Comment:
allow vers override too?[this example](https://docs.microsoft.com/en-us/azure/storage/files/storage-how-to-use-files-linux) uses vers=2.1 (not good idea though).

Reply:
I have added `vers` as a mount option